cli/src/electron/add.ts

```diff
@@ -1,11 +1,18 @@
 import { exec } from 'child_process';
 import { Config } from '../config';
 import { copyTemplate, hasYarn, installDeps, runTask } from '../common';
+import { readFileSync, writeFileSync } from '../util/fs';
+import * as path from 'path';
 
 export async function addElectron(config: Config) {
 
   await runTask(`Adding Electron project in: ${config.electron.platformDir}`, async () => {
-    return copyTemplate(config.electron.assets.templateDir, config.electron.platformDir);
+    const copyReturn = await copyTemplate(config.electron.assets.templateDir, config.electron.platformDir);
+    const capConfigName = JSON.parse(readFileSync(path.join(config.electron.platformDir, '../capacitor.config.json')) + '')['appName'];
```

Comment:
There is no need to read and parse, you can just use require to a json file, also don't use relative urls, we can know the root dir with `config.app.rootDir`.
```suggestion
    const capConfigName = require(join(config.app.rootDir, 'capacitor.config.json')).appName;
```